src/decorators/withResources/createCallConfig.js

```diff
@@ -0,0 +1,34 @@
+import stringifyQueryParamValues from './stringifyQueryParamValues'
+
+export default function createCallConfig(
+  defaultConfig,
+  { include, filters, method = 'get', data }
```

Comment:
Could we also move the HTTP methods to constants? I would put them in Uppercase but according to the Axios docs they are lowercase... 😭